src/CommunityVoices/App/Website/Bootstrap/Routes.php

```diff
@@ -37,6 +38,20 @@ public function get()
                     isset($options['method']) ? $options['method'] : []
                 )
             );
+
+            // Add the API route.
+            $routes->add(
+                "api" . $name,
+                new Route(
+                    $appPrefix . '/api' . $options['notation'],
+                    $options['defaults'],
```

Comment:
on this array we want to merge with `$apiDefaults = ['use-api' => true]`:

~~~php
array_merge($options['defaults'], $apiDefaults)
~~~